libs/ui-watt/src/lib/components/modal/watt-modal.component.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+  TemplateRef,
+  ViewChild,
+  ViewEncapsulation,
+} from '@angular/core';
+import { MatDialog, MatDialogConfig } from '@angular/material/dialog';
+import {
+  exhaustMap,
+  ignoreElements,
+  map,
+  mergeWith,
+  Subject,
+  take,
+  tap,
+} from 'rxjs';
+
+export type WattModalSize = 'small' | 'normal' | 'large';
+
+const sizeConfig: Record<WattModalSize, MatDialogConfig> = {
+  small: { width: '36vw', maxHeight: '45vh' },
+  normal: { width: '50vw', maxHeight: '65vh' },
+  large: { width: '65vw', maxHeight: '90vh' },
+};
+
+/**
+ * Component for representing a binary decision in the form of
+ * a modal window that appears in front of the entire content.
+ */
+@Component({
```

Comment:
suggestion:nitpick: add a code block in the comment to show how to import the module.

```suggestion
/**
 * Component for representing a binary decision in the form of
 * a modal window that appears in front of the entire content.
 * Usage:
 * `import { WattModalModule } from '@energinet-datahub/watt';`
 */
@Component({
```

It will appear similar to:

![image](https://user-images.githubusercontent.com/1096332/174812956-03ecabf9-776f-475c-9fde-db5869da5ac3.png)